src/integrationTest/java/org/opensearch/security/PointInTimeOperationTest.java

```diff
@@ -413,10 +410,17 @@ private String createPitForIndices(String... indices) throws IOException {
     /**
     * Deletes all PITs.
     */
+
     public void cleanUpPits() throws IOException {
+
         try (RestHighLevelClient restHighLevelClient = cluster.getRestHighLevelClient(ADMIN_USER)) {
+
+            restHighLevelClient.getAllPits(DEFAULT);
```

Comment:
qq, why add this?

Reply:
This seemingly fixes the test flakiness. I've found that even just having the declaration of `restHighLevelClient.getAllPits(DEFAULT)` before attempting to delete all PITS makes the `listAllPits_positive` integration test succeed as opposed to failing

Reply:
@EduardoCorazon Have we verified the cause of this?